contracts/utils/math/Math.sol

```diff
@@ -401,7 +403,11 @@ library Math {
     function log256(uint256 value, Rounding rounding) internal pure returns (uint256) {
         unchecked {
             uint256 result = log256(value);
-            return result + (rounding == Rounding.Up && 1 << (result << 3) < value ? 1 : 0);
+            return result + (unsignedRoundsUp(rounding) && 1 << (result << 3) < value ? 1 : 0);
         }
     }
+
+    function unsignedRoundsUp(Rounding rounding) internal pure returns (bool) {
```

Comment:
I missed this missing NatSpec. Don't want to block merging but we can create a good first issue or add later.

Reply:
```suggestion
    /**
     * @dev Returns whether a provided rounding mode is considered rounding up for unsigned integers.
     *
     * This is possible because of the order in the {Rounding} enum that locates `Ceil` and `Expand` at
     * odd positions. 
     */
    function unsignedRoundsUp(Rounding rounding) internal pure returns (bool) {
```

Reply:
```suggestion
    /**
     * @dev Returns whether a provided rounding mode is considered rounding up for unsigned integers.
     */
    function unsignedRoundsUp(Rounding rounding) internal pure returns (bool) {
```